src/CommunityToolkit.Mvvm.Fixers/FieldReferenceForObservablePropertyFieldFixer.cs

```diff
@@ -0,0 +1,70 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using CommunityToolkit.Mvvm.SourceGenerators;
+using CommunityToolkit.Mvvm.SourceGenerators.Diagnostics;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace CommunityToolkit.Mvvm.Fixers;
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+[ExportCodeFixProvider(LanguageNames.CSharp)]
+public class FieldReferenceForObservablePropertyFieldFixer : CodeFixProvider
```

Comment:
Does this have to be public?

Reply:
Would need to add ivts to the test assemblies.

Reply:
It's fine, all other generators, analyzers and suppressors are also public as well anyway 🙂